conans/client/graph/spdx.py

```diff
@@ -0,0 +1,58 @@
+
+def generate_sbom(conanfile, **kwargs):
```

Comment:
We need to align on the interface, if receiving a conanfile, a graph (like deployers)